src/pages/Components/Section/ChangePW.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import { ReactComponent as Prev } from 'assets/svg/prev-icon.svg';
+import warning from 'assets/svg/warning.svg';
+import style from './SearchId.module.scss';
+import ModalForPW from './ModalForPW';
+
+export default function ChangePW() {
+  const pattern = /^.*(?=^.{2,16}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%^&+=]).*$/;
+  const [pw, setPw] = useState<string>('');
```

Comment:
이런 정적인 요소들은 컴포넌트 밖이나, 아예 새로운 모듈로 만들어서 import 해올 수 있을것 같아요.

```suggestion
const pattern = /^.*(?=^.{2,16}$)(?=.*\d)(?=.*[a-zA-Z])(?=.*[!@#$%^&+=]).*$/;

export default function ChangePW() {  
  const [pw, setPw] = useState<string>('');
```

Reply:
오 좋은 정보 참고하겠습니다